src/components/table/DataTable.styled.tsx

```diff
@@ -49,4 +58,9 @@ export const Wrapper = styled("div")({
   gap: 20,
   justifyContent: "flex-end",
   paddingBottom: 15,
+  "@media (max-width: 730px)": {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-end",
+  },
 });
```

Comment:
Ik zie dat je in deze file overal `camelCase` css properties gebruikt. Hoewel dit wel werkt, is het niet de correcte notatie voor web. Native gebruikt `camelCase`, web gebruikt `kebab-case` als conventie. Ook is het aan te raden om units te gebruiken, bijv. `padding-bottom: 15px;`.

Hier een voorbeeld van de conventionele notering van styled-components:
```
export const Wrapper = styled.div`
  align-items: center;
  display: flex;
  gap: 20px;
  justify-content: flex-end;
  padding-bottom: 15px;

  @media screen and (max-width: 730px) {
    align-items: flex-end;
    display: flex;
    flex-direction: column;
  }
`;
```